test-resources/test/test-util/environment-mocks.ts

```diff
@@ -64,7 +65,8 @@ export const destinationBindingCertMock: Service = {
     uri: destinationServiceUri,
     url: providerXsuaaUrl,
     certurl: providerXsuaaCertUrl,
-    uaadomain: uaaDomain
+    uaadomain: uaaDomain,
+    xsappname: 'myapp'
```

Comment:
Is a [mandatory property ](https://github.wdf.sap.corp/CPSecurity/node-xs2sec/blob/07a3b01159e82a4207fd50250e83571a353a1782/src/service/XsuaaService.js#L42)